Permission path delimiter (#48944)
nodejs/node: doc/api/permissions.md

* `--allow-fs-read=/home/test*` will allow read access to everything
  that matches the wildcard. e.g: `/home/test/file1` or `/home/test2`

##### Accessing files with comma in path

> To access files with comma in path you can change the path delimiter using the

Reviewer: ```suggestion
To access files with a comma in the path you can change the path delimiter using the
```

Author: This will need line changes not to exceed 80 characters, I will change it

`--permission-fs-path-delimiter` flag to set a value not used in any of the
paths you want to access.

```console
$ node --experimental-permission --allow-fs-read="/with,commas_/home" \
--permission-fs-path-delimiter=_ index.js
```

> Note when using bash special characters like `;` escape or quoting is required.

Reviewer: ```suggestion
Note when using special shell characters such as `;` escaping or quoting is required.
```
